addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3142,8 +3142,8 @@ var SnippetsMenu = Widget.extend({
         const docSelection = document.getSelection();
         const $currentSelectionTarget = docSelection && docSelection.rangeCount > 0 ? $(docSelection.getRangeAt(0).commonAncestorContainer) : $();
         // Do not  toggle visibility if the target is inside the toolbar ( eg. during link edition).
-        if ($currentSelectionTarget.parents('#toolbar').length ||
-            (e && $(e.target).parents('#toolbar').length)
+        if ($currentSelectionTarget.parents('#toolbar, #o_we_editor_toolbar_container').length ||
+            (e && $(e.target).parents('#toolbar, #o_we_editor_toolbar_container').length)
```

Comment:
- the toolbar is not inside the toolbar_container? :thinking: 
- Also I know it was like that before, but should not this use closest instead of parents?

Reply:
- the toolbar is not inside the toolbar_container? -> yes,indeed. 
I probably thought that this code was used when the toolbar is not in the toolbar_container.

- $currentSelectionTarget is always a child of  the toolbar_container so I guess it's ok. It works well like that but I let you choose what we do

Reply:
Actually I do not understand why we check the selection and in addition the target of the event...

Reply:
Yes, there is a difference between closest and parents as closest searches starting from the element itself instead of its parent but if we want to search for an ancestor starting from the parent, it is still better to do .parent().closest() then (or its JS equivalent) as parents keeps searching all ancestors up to the body. It's nearly never what we want. So here use .closest() or .parent().closest() depending on which is right, but not `parents`.